pkg/utils/validator/clusterresourceplacement.go

```diff
@@ -213,6 +225,44 @@ func validateClusterAffinity(clusterAffinity *placementv1beta1.ClusterAffinity,
 	return apiErrors.NewAggregate(allErr)
 }
 
+func validateTolerations(tolerations []placementv1beta1.Toleration) error {
+	allErr := make([]error, 0)
+	tolerationMap := make(map[placementv1beta1.Toleration]bool)
+	for _, toleration := range tolerations {
+		switch toleration.Operator {
+		case corev1.TolerationOpExists:
+			if toleration.Key != "" {
+				for _, msg := range validation.IsQualifiedName(toleration.Key) {
```

Comment:
Reusing validation logic for toleration key and value from k8s https://github.com/kubernetes/kubernetes/blob/7080b51ee92f67623757534f3462d8ae862ef6fe/pkg/apis/core/validation/validation.go#L3908-L3930